### PR TITLE
Add place deps for fused_all_reduce_op_handle

### PR DESCRIPTION
**Pull request by chengduoZH**
Add place deps for fused_all_reduce_op_handle.
image

### CODE REVIEW

**Review by chenwhql — COMMENTED** [review]
image

I try to compiled with this branch, still failed.

**Comment by chenwhql** [comment]
> image
> 
> I try to compiled with this branch, still failed.

It's my problem, remove build dir, then recompiled, success

File: paddle/fluid/framework/details/CMakeLists.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ if(WITH_GPU)
     nv_library(all_reduce_op_handle SRCS all_reduce_op_handle.cc DEPS op_handle_base scope lod_tensor ddim memory
             dynload_cuda variable_visitor)
     nv_library(fused_all_reduce_op_handle SRCS fused_all_reduce_op_handle.cc DEPS op_handle_base scope lod_tensor ddim memory
-            dynload_cuda variable_visitor)
+            dynload_cuda variable_visitor place)
```
Comment (chenwhql) [inline paddle/fluid/framework/details/CMakeLists.txt:28]: but I remove these two `place`, it also works

Comment (luotao1) [reply paddle/fluid/framework/details/CMakeLists.txt:28]: 可能是因为你有缓存在，可以把整个build目录删了，再试试去掉place是否正常。

Comment (chenwhql) [reply paddle/fluid/framework/details/CMakeLists.txt:28]: delete build directory then recompile, still work

image

Comment (luotao1) [reply paddle/fluid/framework/details/CMakeLists.txt:28]: @chenwhql 要不先merge这个PR，现在会阻碍一部分同学编译。

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**